solutions/standard/variables.tf

```diff
@@ -11,78 +11,99 @@ variable "region" {
 
 variable "prefix" {
   type        = string
-  description = "A unique identifier for resources. Must begin with a lowercase letter and end with a lowercase letter or number. This prefix will be prepended to any resources provisioned by this template."
+  description = "Optional. The prefix to append to all resources that this solution creates. Must begin with a lowercase letter and end with a lowercase letter or number."
 }
 
-variable "resource_group" {
+variable "resource_group_name" {
   type        = string
-  description = "An existing resource group name to use for this example. If not specified, a new resource group is created."
-  default     = null
+  description = "The name of a new or the existing resource group to provision the client to site VPN. If a prefix input variable is passed, it is prefixed to the value in the `<prefix>-value` format."
 }
 
-variable "base_vpn_gateway_name" {
+variable "use_existing_resource_group" {
+  type        = bool
+  description = "Whether to use an existing resource group."
+  default     = false
+}
+
+variable "name" {
   type        = string
   description = "The name of the VPN."
   default     = "vpn"
 }
 
-##############################################################
-# Secret Manager
-##############################################################
+##############################################################################
+# Secrets Manager resources
+##############################################################################
 
-variable "sm_service_plan" {
+variable "existing_secrets_manager_instance_crn" {
   type        = string
-  description = "The service/pricing plan to use when provisioning a new Secrets Manager instance. Allowed values: `standard` and `trial`. Only used if `provision_sm_instance` is set to true."
-  default     = "standard"
+  description = "The CRN of existing secrets manager to use to create service credential secrets."
```

Comment:
I think the description is wrong here - it mentions service credential secrets when thats not what this is used for

Reply:
updated